USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -8,10 +8,34 @@ namespace USFMToolsSharp
     public class HtmlRenderer
     {
         public List<string> UnrenderableTags;
+        public List<string> FootnoteTextTags;
+
+        private bool isSingleSpaced = true;
+        private bool hasOneColumn = true;
+        private bool isL2RDirection = true;
+        private bool isTextJustified = false;
+        private bool separateChapters = false;
+
+        public string FrontMatterHTML { get; set; }
+        public string InsertedFooter { get; set;}
+        public string InsertedHead { get; set; }
 
         public HtmlRenderer()
         {
             UnrenderableTags = new List<string>();
+            FootnoteTextTags = new List<string>();
+        }
+        public HtmlRenderer(bool isSingleSpaced=true, bool hasOneColumn=true, bool isL2RDirection=true, bool isTextJustified=false, bool separateChapters=false)

```

Comment:
I don't think some of these options should be booleans.  Spacing can be 1x, 1.5x, 2x, or more. Users may want 1, 2, or 3 columns.  Text align can be left, right, center, or justified.

Reply:
Since these are just setting CSS classes in the output text, I wonder if we should just pass in a set of strings that are the css classes to apply to the divs.

Reply:
What do you think @rbnswartz ?

Reply:
So, if the user had more options, I would have to programmatically configure the style.css. 
Is that what you were thinking @PurpleGuitar ?

Reply:
The Unit Tests are located in the other repository, should I bring it to this repository?
 [Unit Tests for USFMToolsSharp](https://github.com/WycliffeAssociates/quick_usfm_converter/tree/cro_sprint_1/USFMToolsSharpTest)

Reply:
@BryanHo10 Yes they should be in the repository with their code. Create another branch and PR with that.